rocAL/rocAL/include/meta_data/meta_node.h

```diff
@@ -33,7 +33,7 @@ class MetaNode
 public:
     MetaNode() {}
     virtual ~MetaNode() {};
-    virtual void update_parameters(MetaDataBatch* input_meta_data) = 0;
+    virtual void update_parameters(MetaDataBatch* input_meta_data, bool segmentation) = 0;
```

Comment:
can we use const  bool segmentation as this value is not going to be modified

Reply:
Resolved in the latest commit